models/nulog/masker.py

```diff
@@ -28,12 +28,27 @@ def __init__(
         self.remove_delimiters = r'([| \(|\)|\[|\]\'|\{|\}|"|,])'
         self.ansi_escape = re.compile(r"(\x9B|\x1B\[)[0-?]*[ -\/]*[@-~]")
 
-    def mask(self, content: str):
+    def mask(self, content: str, is_control_plane_log: bool):
 
         # get rid of escape keys
         content = self.ansi_escape.sub("", content)
 
-        # reduce any json objects (TODO)
+        # If log is a control plane log, mask out content from structured control plane log messages which follow the 'a="somecontent" to be a=<a>'
+        if is_control_plane_log:
+            assignment_regex = '([A-z]+)="(.*?)"'
```

Comment:
`A-z` is improper regex - the character sets A-Z and a-z are not contiguous in the ascii table. If you want to match uppercase and lowercase letters you need `a-zA-Z`

Reply:
Hmm, so I tried this regex out again just now to be sure and it is working intended for upper or lower case patterns.

Reply:
It will match upper and lower case letters. It will also match everything between Z and a.

Reply:
Yeah, but would those appear in that format specifically within control plane log messages?

Reply:
Let me look into this some more.